Source/WebKit/UIProcess/UserMediaProcessManager.cpp

```diff
@@ -86,7 +87,7 @@ bool UserMediaProcessManager::willCreateMediaStream(UserMediaPermissionRequestMa
 
     bool needsAppleCameraSandboxExtension = needsVideoSandboxExtension && needsAppleCameraService();
     if (needsAppleCameraSandboxExtension) {
-        extensionCount++;
+        extensionCount += 2;
```

Comment:
Why is this a +2, while the others are only +1? I'm guessing you know something about AppleCameraSandboxExtension requiring a mach bootstrap extension, and the other extensions not requiring it? Is there some way we can make this more robust, so that this number will continue to be right even if AppleCameraSandboxExtension stops requiring mach bootstrap? (And what if one of the others starts requiring it?)

Reply:
Yes, the AppleCameraSandboxExtension is a Mach lookup extension, so a Mach bootstrap extension is needed in this case. Only one bootstrap extension is required, even if there are man Mach lookup extensions. I have attempted to make this more clear and robust in the latest patch by creating a separate bootstrap extension data member in the latest patch.

Thanks for reviewing!